docs/readthedocs/source/doc/Chronos/Howto/how_to_use_benchmark_tool.md

```diff
@@ -0,0 +1,135 @@
+# Use Chronos benchmark tool
+This page demonstrates how to use Chronos benchmark tool to benchmark forecasting performance on platforms.
+
+## Basic Usage
+The benchmark tool is installed automatically when Chronos is installed using conda. To get preliminary knowledge of forecasting performance on platform, run benchmark tool with default options using following command:
```

Comment:
The benchmark tool is installed automatically when `bigdl-chronos` is installed. To get information about performance (currently for forecasting only) on the your own machine.

Run benchmark tool with default options using following command: